packages/quantic/force-app/main/default/lwc/quanticSort/e2e/fixture.ts

```diff
@@ -0,0 +1,112 @@
+import {SortObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+
+const pagerUrl = 's/quantic-sort';
+
+interface SortOptions {}
+
+type QuanticSortE2EFixtures = {
+  sort: SortObject;
+  sortCustom: SortObject;
+  sortInvalid: SortObject;
+  search: SearchObject;
+  options: Partial<SortOptions>;
+};
+
+type QuanticSortE2ESearchFixtures = QuanticSortE2EFixtures & {
+  urlHash: string;
+};
+
+type QuanticSortE2EInsightFixtures = QuanticSortE2ESearchFixtures & {
```

Comment:
The Insight Fixture should contain `QuanticSortE2EFixtures ` + `insightSteup` and not `QuanticSortE2ESearchFixtures ` + `insightSetup`

You can follow the following example: https://github.com/coveo/ui-kit/blob/7a21e8674d66142d14980f1efa360b59d801ee35/packages/quantic/force-app/main/default/lwc/quanticTab/e2e/fixture.ts#L15-L27.

Reply:
I will update this. Do you want I update pager too? as I followed your example on Pager component?